Installer/License.txt

```diff
@@ -0,0 +1,90 @@
+Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International Public License
```

Comment:
Can we add this text to the top of this file to be displayed in the agreement window?
```
This is a human-readable summary of (and not a substitute for) the full license below.  

Under the following terms:  
    Attribution — You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.  
    NonCommercial — You may not use the material for commercial purposes.  
    NoDerivatives — If you remix, transform, or build upon the material, you may not distribute the modified material.  
    No additional restrictions — You may not apply legal terms or technological measures that legally restrict others from doing anything the license permits.
```